xchange-poloniex/src/main/java/org/knowm/xchange/poloniex/PoloniexAdapters.java

```diff
@@ -283,11 +282,11 @@ public static ExchangeMetaData adaptToExchangeMetaData(
   }
 
   private static FundingRecord adaptAdjustment(PoloniexAdjustment a) {
-    FundingRecord.Type type = DEPOSIT;
+    FundingRecord.Type type = OTHER_INFLOW;
```

Comment:
could we detect deposits instead of falling back on OTHER_INFLOW for anything other then an airdrop?

Reply:
Poloniex return type consists out of three different objects:
`{
	"adjustments": [ ... ],
	"deposits": [ ... ],
	"withdrawals": [ ... ]
}`

If their naming scheme is consistent adjustments can't be deposits. That's way I choose `OTHER_INFLOW`.